ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/event/ng/EditorFileStatusNotificationOperation.java

```diff
@@ -48,22 +49,26 @@
     private NotificationManager notificationManager;
 
     @Inject
-    public EditorFileStatusNotificationHandler(EventBus eventBus,
-                                               DeletedFilesController deletedFilesController,
-                                               AppContext appContext) {
-        super(VfsFileStatusUpdateDto.class, Void.class);
+    public EditorFileStatusNotificationOperation(EventBus eventBus, DeletedFilesController deletedFilesController, AppContext appContext) {
         this.eventBus = eventBus;
         this.deletedFilesController = deletedFilesController;
         this.appContext = appContext;
+    }
 
+    @Inject
+    public void configureHandler(RequestHandlerConfigurator configurator) {
```

Comment:
Yeah, as you said as a general practice we use constructor injection to initialize class fields, in this method we do not initialize class fields, instead we do post construct initialization, so it looks quite appropriate here.